microshift_running_apps/microshift-operators-olm.adoc

```diff
@@ -0,0 +1,64 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="olm-with-microshift"]
+= Using Operator Lifecycle Manager with {microshift-short}
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-operators-olm
+
+toc::[]
+
+OLM is used in {microshift-short} for installing and running optional link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/architecture/control-plane#olm-operators_control-plane[add-on Operators].
+
+* Cluster Operators as applied in {ocp} are not used in {microshift-short}.
+* You can add your own catalogs for the Operators you want to use with your applications.
+* You must use the CLI to conduct OLM activities with {microshift-short}. The console and OperatorHub GUIs are not available.
+* You must configure the system to use a mirror by updating your CRI-O configuration directly because `ImageContentSourcePolicy` is not supported in {microshift-short}. See link:https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/{product-version}/html/installing/microshift-deploy-with-mirror-registry#microshift-configuring-hosts-for-mirror_microshift-deployment-mirror[Configuring hosts for mirror registry access] for details.
+* Install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-version}/html/cli_tools/opm-cli#cli-opm-install[`opm` CLI]
+
+[IMPORTANT]
+====
+Before using an Operator, verify with the provider whether the Operator is supported on {product-title}.
+====
+
+This section of the docs is definitely referring to adding olm based operators to a disconnected cluster. I don't have a strong opinion about whether we need an entirely separate copy of the documentation, but the procedure to actually do the mirroring is definitely what we point folks to regardless of whether or not they'll have access to the openshift console. But yes, we should probably firm up our information around how to mirror images without ICSPs if we're not expecting users who already interact with microshift to know how to modify the container runtime bits directly.
```

Comment:
This looks like content of one of the comments in this PR

Reply:
Yes, that's an oops. Must have been in my clipboard and I dropped it by accident.